db/model/event.js

```diff
@@ -8,7 +8,7 @@
  */
 
 /**
- * db/model/event.js
+ * db/model/events.js
```

Comment:
POSTing to bulk upsert follows the same code path as posting to /v1/samples/upsert so I didn't think it was completely necessary (and it adds extra complication to the testing, too, so I just punted)... I can add those but it will take some time